src/app/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="timesheet">
```

Comment:
The component is named `header` but you have `timesheet` class everywhere.
If the component is only for the timesheet you can name it `timesheet-header` maybe ?
Or the class here is `header` and not timesheet ?